gammapy/spectrum/tests/test_models.py

```diff
@@ -150,6 +151,14 @@ def table_model():
         eflux_1_10TeV=u.Quantity(6.41406327, "TeV cm-2 s-1"),
         e_peak=np.nan * u.TeV,
     ),
+    dict(
+        name="Gaussian",
+        model=Gaussian(norm=2 / u.cm ** 2 / u.s, mean=4 * u.TeV, sigma=3 * u.TeV),
```

Comment:
I would also/rather test a gaussian centered at 2 TeV with a very thin width to check.

Reply:
why do you want a very thin width ?

Reply:
Because it is one of the test case where we might better know the expected results, and this is also the situation when the analytical integration is important